src/einsteinpy/symbolic/tensor.py

```diff
@@ -180,3 +181,123 @@ def simplify(self):
 
         """
         return sympy.simplify(self.tensor())
+
+
+class GRTensor(Tensor):
```

Comment:
Okay, I change my words, rename it to `BaseRelativityTensor` or something like that!

Reply:
Correct! Doing it!